### DISCUSSION TITLE
Feature request: Filtering initial value templates when creating documents in place

### DISCUSSION THREAD

**Discussion by ehuelsda** [sanity-io/sanity#3806] [Sanity Studio Feedback]
As far as I've been able to find out, there is currently no way (in v2 nor v3), to be able to filter which (or if) initial value templates should be possible to choose from when creating documents in place. To simplify the editing experience, I think that would have been very useful to be able to do. 

Say I have an employee document schema and several initial value templates (e.g. developer, designer ...) defined for that schema in my sanity.config.ts. Also I have a department schema where I have an array of references to employee. When hitting the 'create new' button on the reference field I get to choose between creating a new employee and all the different initial value templates. What I want though, is to only create developers in this array.

**Comment by cultura-saatio** [comment]
Run into the same issue. Need to be able to determine initial value when create new item from reference field.
